examples/read_sensor.py

```diff
@@ -0,0 +1,41 @@
+import threading
+import time
+
+from CreaTeBME import connect
+import asyncio
+
+sensors = []
+
+
+async def connect_sensors():
+    sensors.extend(await connect())
+    print(sensors)
+
+
+    # while True:
+    #     await asyncio.sleep(1)
+
+def worker():
+    loop = asyncio.new_event_loop()
+    task = loop.create_task(connect_sensors())
+    loop.run_forever()
+
+
+def print_measurement(data):
+    print('het werkt:', data)
```

Comment:
```suggestion
    print('Sensor data:', data)
```
Change this to be English